code/rounds/MapSelectionRound.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Sandbox;
+
+using TTTReborn.Globals;
+using TTTReborn.Map;
+using TTTReborn.Player;
+using TTTReborn.Settings;
+
+namespace TTTReborn.Rounds
+{
+    public class MapSelectionRound : BaseRound
+    {
+        public override string RoundName => "Map Selection";
+        public override int RoundDuration
+        {
+            get => ServerSettings.Instance.Round.MapSelectionRoundTime;
+        }
+
+        protected override void OnTimeUp()
+        {
+            base.OnTimeUp();
+
+            IDictionary<long, string> playerIdMapVote = Gamemode.Game.Instance?.MapSelection?.PlayerIdMapVote;
+            IDictionary<string, int> mapToVoteCount = MapSelectionHandler.GetTotalVotesPerMap(playerIdMapVote);
+            if (mapToVoteCount.Count == 0)
+            {
+                Global.ChangeLevel(ServerSettings.Instance?.Map?.DefaultMap ?? "facepunch.flatgrass");
```

Comment:
Do we want to random if there are no votes? or just throw it to the default map. @Alf21